netket/graph/custom_graph.py

```diff
@@ -0,0 +1,99 @@
+from abstract_graph import PyAbstractGraph
+
+import numpy as _np
+import networkx as _nx
+
+class PyCustomGraph(PyAbstractGraph):
+	r"""A custom graph, specified by a list of edges and optionally colors."""
+
+	def __init__(self, edges=[]):
+		"""
```

Comment:
please have a look at CONTRIBUTING.md in the branch 3.0 (in order to have consistent coding style in all the codebase we adopt black coding style)

Reply:
I'll do that